package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fcp-fd-data",
-  "version": "0.1.6",
+  "version": "0.1.7",
```

Comment:
Just thinking this PR contains a good chunk of really cool and new GQL functionality. In fairness I haven't seen a lot of the PRs previous to this one but feels like adding the ability to query our database via GQL is maybe a minor update rather than a patch? (Lessons learned from my own previous mistake really so happy to be corrected here 😅).

Reply:
yeah, will bump to 0.2.0, fair enough it's 30+ file changes